scripts/release.py

```diff
@@ -206,16 +206,236 @@ def create_draft_release(
     print_release_notes = bool(os.getenv("PRINT"))
     if print_release_notes:
         print(
-            """\nName:%s\nTag:%s\nprerelease:%s\ntarget_commitish:%s\nmessage:%s
+            """RELEASE NOTES INFO:\nName:%s\nTag:%s\nprerelease:%s\ntarget_commitish:%s\nmessage:%s
               """
             % (name, tag, prerelease, base_branch, rn)
         )
     else:
         dd_repo.create_git_release(
             name=name, tag=tag, prerelease=prerelease, draft=True, target_commitish=base_branch, message=rn
         )
-        print("Please review your release notes draft here: https://github.com/DataDog/dd-trace-py/releases")
+        print("\nPlease review your release notes draft here: https://github.com/DataDog/dd-trace-py/releases")
+
+    return name, rn
+
+
+def setup_gh():
+    # get dd-trace-py repo
+    gh_token = os.getenv("GH_TOKEN")
+    if not gh_token:
+        raise ValueError(
+            "We need a Github token to generate the release notes. Please follow the instructions in the script."
+        )
+    g = Github(gh_token)
+    return g.get_repo(full_name_or_id="DataDog/dd-trace-py")
+
+
+def create_notebook(dd_repo, name, rn, base, rc, patch):
+    dd_api_key = os.getenv("DD_API_KEY_STAGING")
+    dd_app_key = os.getenv("DD_APP_KEY_STAGING")
```

Comment:
should also check if these are provided, otherwise we get no indication that something went wrong until a keyerror happens in the returned data